.github/workflows/build.yml

```
@@ -39,6 +39,8 @@ jobs:
           fetch-depth: 0
       - name: 'Install .NET Core SDK'
         uses: actions/setup-dotnet@4d6c8fcf3c8f7a60068d26b594648e99df24cee3 # v4.0.0
+        with:
+          dotnet-version: 6.0.x
```

I think we need to install multiple versions here

```suggestion
          dotnet-version: |
            6.0.x
            8.0.x
```

We build using the .NET 8.0 SDK, but target the .NET 6.0 runtime. And if we don't specify `dotnet-version` the `actions/setup-dotnet` action uses the version in `global.json` (Currently .NET 8.0).

---

Makes sense!